build.sbt

```diff
@@ -46,9 +41,31 @@ lazy val microservice = Project(appName, file("."))
     IntegrationTest / testGrouping := oneForkedJvmPerTest((IntegrationTest / definedTests).value),
     IntegrationTest / parallelExecution := false
   )
-  .settings(silencerSettings)
   .disablePlugins(JUnitXmlReportPlugin) //Required to prevent https://github.com/scalatest/scalatest/issues/1427
 
+lazy val commonSettings = List(
+  majorVersion := 0,
+  scalaVersion := "2.13.8",
+  scalacOptions ++= scalacFlags,
+  retrieveManaged := true,
+  dependencyOverrides += "commons-codec" % "commons-codec" % "1.15",
+  libraryDependencies ++= AppDependencies.compile ++ AppDependencies.test
+)
+
+lazy val scalacFlags = List(
+  "-deprecation",                                // warn about use of deprecated APIs
+  "-encoding", "UTF-8",                          // source files are in UTF-8
+  "-feature",                                    // warn about misused language features
+  "-language:implicitConversions",
+  "-unchecked",                                  // warn about unchecked type parameters
+  //"-Wconf:any:warning-verbose",
```

Comment:
left in comment?

Reply:
No. Just left as a future note in case of warnings => errors tricky to solve